### ISSUE TITLE
Any `if` condition after a Callable being defined on an object property throws an error in the Script Editor

### ISSUE THREAD

**Issue by henriquelalves** [godotengine/godot#78828]
### Godot version

4.0.3
EDIT: Just confirmed this error exists in 4.1 Beta 3 too.

### System information

MacBook Ventura 13.4

### Issue description

It's easier to understand the error with a script example:

```
func _ready():
	var object_with_callable = ObjectWithCallable.new()
	
	# This works
	var a = func():
		pass
	if true:
		pass
	
	# This doesn't
	object_with_callable.callable = func():
		pass
	if true:
		pass


class ObjectWithCallable:
	var callable: Callable
```

With this script, Godot will throw an error on the last `if true:` line, with: `Error at(x,y): Expected "else" after ternary operator condition`.

### Steps to reproduce

Just define a callable to an object property (doesn't need to be typed), and add an `if` condition on the next line, per the example.

### Minimal reproduction project

Any project with the lines above will throw this error.

EDIT: Just to be safe, this is a project with the same lines in the example:
[CallableProperyBug.zip](https://github.com/godotengine/godot/files/11903326/CallableProperyBug.zip)

**Comment by dalexeev** [comment]
Probably duplicate of #73273. Please check 4.1 RC 1.

**Comment by henriquelalves** [comment]
Yup, it's fixed on 4.1 RC 1. Closing the issue.
